.gitignore

```diff
@@ -0,0 +1,7 @@
+.idea/misc.xml
```

Comment:
I think environment-specific files are better in a global .gitignore (https://gist.github.com/subfuzion/db7f57fff2fb6998a16c).